### ISSUE TITLE
기타 실행시 IE를 종료하는 게 내부적으로있나요?

### ISSUE THREAD

**Issue by oops0905** [naver/guitar#394]
run("C:\GUITAR\BIN\GUITAR.EXE C:\GUITAR\DATA\A.txt") 
이렇게 실행되도록 exe파일을 만들어서 실행하면
띄워져있던 IE창이 사라지고 기타가 실행되고 A스크립트가 돌아갑니다.

run("C:\GUITAR\BIN\GUITAR.EXE")
run("C:\GUITAR\DATA\A.txt") 
이렇게 run을 2번하면 띄워져있던 IE창은 유지되고 기타는 실행되지만 A스크립트는 실행되지 않습니다.

결국 원하는것은 띄워져있던 IE창이 종료안되고 유지되면서 exe파일 클릭하면 기타가 실행되고 A스크립트가 실행되게 하는것인데 방법이 있을까요?

**Comment by ssmmhh** [comment]
네, commandline 으로 guitar를 실행할 경우 자동으로 기존에 열려진 모든 브라우저를 종료하는 기능을 가지고 있습니다.
(열려진 브라우저 때문에 예기치 않은 작동이 발생할수 있기 때문입니다.)

GUITAR.INI 옵션에서 이 기능을 수행하지 않도록 가능합니다.

CloseAllBrowser = False 

로 변경하여 사용하시면 됩니다.
